kotlin/src/main/AndroidManifest.xml

```diff
@@ -1 +1,17 @@
-<manifest package="app.rive.runtime.kotlin" />
\ No newline at end of file
+<manifest package="app.rive.runtime.kotlin"
```

Comment:
Oh! Initializer's back in the manifest! Does this mean we no longer have to include it in the example app Manifest?